lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -112,6 +106,20 @@ func (i *InteractionHandlersImpl) BindHandler(params interaction.InteractionBind
 		)
 	}
 
+	// Create a new attach server.  If one already exist, stop it first.  Do this
+	// because we can't assume unbind was called after the previous bind.  Test
+	// for this is compose up, ctrl-c, then compose up again.
+	if i.attachServer != nil {
```

Comment:
I thought bind was a container level operation, while the attach server is shared by all containers? Wouldn't this affect containers that are already connected to the attach server?